docs/topics/proc_deploying-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,30 @@
+[id='deploying-application-to-standalone-rhel_{context}']
+= Deploying an application to standalone Red Hat Enterprise Linux
+
+You can deploy a {VertX} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* Red Hat Enterprise Linux 7
+* OpenJDK 8 installed
+* Maven 3.5 or later installed
+
+.Procedure
+
+. Download the {VertX} runtime from the link:https://maven.repository.redhat.com/[Red Hat Maven repository] to your RHEL 7 system.
+
+. Create a project for your application. Consider packaging your application as a fat jar to include all of the necessary dependencies.
+
+. Start your application:
++
+ifdef::built-for-vertx[]
+[source,bash,options="nowrap",subs="attributes+"]
+----
+java -jar target/my-app-fat.jar
+----
+endif::built-for-vertx[]
+
+.Verification steps
+ifdef::built-for-vertx[]
```

Comment:
If the curl command is always the same (IIRC it is), there's probably little use in conditionalising this step.